src/api/lib/tasks/databases.rake

```diff
@@ -1,3 +1,5 @@
+require_relative '../../app/models/application_record'
```

Comment:
Huh?

Reply:
If I dont include this then I get `uninitialized constant ApplicationRecord` when I run the rake task from the command line. I don't really understand it either..